tests/integration_tests/framework/docker.py

```diff
@@ -227,28 +213,36 @@ def _set_container_id_and_ip(container_details):
 
 
 def _retry(func, exceptions, cleanup=None):
-    for _ in range(600):
+    err = None
+    for _ in range(2600):
```

Comment:
this is to be changed but we currently made rabbitmq take a loooong time to start